features/step_definitions/volunteer_op_steps.rb

```diff
@@ -22,4 +22,19 @@
   click_on 'Edit'
   fill_in('Description', :with => description)
   click_on 'Update a Volunteer Opportunity'
+end
+
+Given /^I should see (\d+) markers in the map$/ do |num|
+  expect(page).to have_css('.vol_op', count: num)
+end
+
+Then(/^I should see a link to "(.*?)" page "(.*?)"$/) do |link, url|
+  page.should have_link(link, :href => url)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)